vault.go

```diff
@@ -24,8 +24,14 @@ func getCredentialSetFromVault(host string) (map[string]string, error) {
 		return nil, errNoData
 	}
 
+	var data map[string]interface{}
 	var out = map[string]string{}
-	for k, v := range secret.Data {
+	var ok bool
+	data, ok = secret.Data["data"].(map[string]interface{})
+	if !ok {
+		data = secret.Data
+	}
+	for k, v := range data {
```

Comment:
Please simplify the three variable declarations in a single `var ( ... )` block and use shorter `if data, ok = ...; !ok {` syntax for condition

Reply:
ok, updated, plz review tks